lib/rubygems/remote_fetcher.rb

```diff
@@ -104,14 +104,14 @@ def initialize(proxy=nil, dns=nil, headers={})
   # Should probably be integrated with #download below, but that will be a
   # larger, more encompassing effort. -erikh
 
-  def download_to_cache(dependency)
+  def download_to_cache(dependency, install_dir = Gem.dir)
```

Comment:
Personal preference, but maybe change `install_dir` to a kwarg?  They stick out a little more when used...

Reply:
I don't want to make it inconsistent with the above `download` method.